### PR TITLE
[ISSUE-388][SHELL] Add restart script for worker's quick recover

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: sbin/rss-daemon.sh [hunk 1]
```diff
@@ -181,6 +181,27 @@ case $option in
...
+
+    if [ -f $pid ]; then
+      TARGET_ID="$(cat "$pid")"
+      if [[ $(ps -p "$TARGET_ID" -o comm=) =~ "java" ]] || [[ $(ps -p "$TARGET_ID" -o comm=) =~ "jboot" ]]; then
+        echo "stopping $command"
+        kill "$TARGET_ID" && rm -f "$pid"
+        while [[ $(ps -p "$TARGET_ID" -o comm=) != "" ]];
+        do
+          echo "waiting worker graceful shutdown, sleep 1s"
+          sleep 1s
```
Comment (waitinfuture) [inline sbin/rss-daemon.sh:194]: Shall we add timeout here?

File: sbin/rss-daemon.sh [hunk 2]
```diff
@@ -181,6 +181,27 @@ case $option in
...
+        done
+        run_command class "$@"
+      else
+        echo "no $command to restart"
+      fi
+    else
+      echo "no $command to restart"
```
Comment (waitinfuture) [inline sbin/rss-daemon.sh:201]: Shall we just start worker if no existing worker?

File: sbin/rss-daemon.sh (hunk 1 above)
Comment (AngersZhuuuu) [reply sbin/rss-daemon.sh:194]: > Shall we add timeout here?

I think we have a timeout in graceful shutdown, so don't need to do this here?

File: sbin/rss-daemon.sh (hunk 2 above)
Comment (AngersZhuuuu) [reply sbin/rss-daemon.sh]: > Shall we just start worker if no existing worker?

How about current

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture How about current?

File: sbin/rss-daemon.sh (hunk 1 above)
Comment (waitinfuture) [reply sbin/rss-daemon.sh:194]: In case something wrong with graceful shutdown, then restart script will hang forever

Comment (AngersZhuuuu) [reply sbin/rss-daemon.sh:196]: > 

DOne

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**